docs/RELEASE_WORKFLOW.md

```
@@ -168,21 +189,35 @@ If you prefer manual control:
+5. Build application (tauri build)
+   ↓
+6. Generate updater signatures for tag releases and main-branch draft releases
+   ↓
+7. Package and upload GitHub Actions artifacts
+   ↓
+8. Stop here for manual artifact builds
+```
+
+### Release Publishing Process
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)